app/services/feature_flag_service/api/features.rb

```diff
@@ -6,29 +6,29 @@ def initialize(feature_flag_store)
       @feature_flag_store = feature_flag_store
     end
 
-    def enable(community_id:, features:)
+    def enable(entity_id:, features:)
```

Comment:
As you said earlier, it's also possible to make this so that enable takes hash with keys: community_id, person_id, and features. So, it's possible to make some kind of per flag-service-type check which accepts only parameters community_id + features or person_id + features combos (depending on service type).